src/material-experimental/mdc-table/table.ts

```diff
@@ -16,7 +16,7 @@ import {
 import {_DisposeViewRepeaterStrategy, _VIEW_REPEATER_STRATEGY} from '@angular/cdk/collections';
 
 @Component({
-  selector: 'table[mat-table]',
+  selector: 'mat-table table[mat-table]',
```

Comment:
Shouldn't it be `mat-table, table[mat-table]`? The current selector is looking for tables inside a `mat-table` element.
Also it seems like there are other directives that are only scoped to table elements, like `MatHeaderCell`, `MatCell` etc. I wonder whether @andrewseguin did this intentionally?

Reply:
Additionally, the element-based APIs will need additional CSS to align with the flexbox-based style of the older version.